scripts/node/pattern_lab.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
+
+shell.echo("Let's install PATTERN LAB.");
+shell.rm('-rf', 'pattern-lab');
+shell.exec('composer create-project -n pattern-lab/edition-twig-standard pattern-lab')
+shell.rm('-rf', 'pattern-lab/source');
+shell.ln('-s', '../components', 'pattern-lab/source')
+shell.exit();
```

Comment:
This is fine, but typically I'd use the core modules, e.g. [fs](https://nodejs.org/docs/latest/api/fs.html) to achieve this sort of thing.